app/scripts/lib/notification-manager.js

```diff
@@ -29,17 +29,30 @@ class NotificationManager {
       // bring focus to existing chrome popup
       await this.platform.focusWindow(popup.id)
     } else {
-      const { screenX, screenY, outerWidth, outerHeight } = window
-      const notificationTop = Math.round(screenY + (outerHeight / 2) - (NOTIFICATION_HEIGHT / 2))
-      const notificationLeft = Math.round(screenX + (outerWidth / 2) - (NOTIFICATION_WIDTH / 2))
+      let left = 0
+      let top = 0
+      try {
+        const lastFocused = await this.platform.getLastFocusedWindow()
+        // Position window in top right corner of lastFocused window.
+        top = lastFocused.top
+        left = lastFocused.left + (lastFocused.width - NOTIFICATION_WIDTH)
+      } catch (_) {
```

Comment:
What error would we be catching here? Is it possible that there isn't a last focused window?

Reply:
If we know that the window in the background doesn't have a size, we could forgo the calculations below.

Reply:
> What error would we be catching here? Is it possible that there isn't a last focused window?

Per @Gudahtt , android chrome doesn’t yet support getLastFocused on the windows api. This is an effort to have a fallback that behaved like the previous iteration.

Reply:
> If we know that the window in the background doesn't have a size, we could forgo the calculations below.

True, though I’d need to test my assumption on chrome android to be sure that it behaves that way as well.

Reply:
Gotcha! (My bad, I didn't look at the existing resolved convos. 😅)